File: src/main/java/com/codecool/hogwartshouses/persistence/entity/Recipe.java
package com.codecool.hogwartshouses.persistence.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.List;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Recipe {
    @Id @GeneratedValue
    private long id;
    private String name;
    @OneToOne
    private Student brewer;
    @OneToMany
    List<Ingredient> ingredients;
}

File: client/src/content-components/potions/brewingSection/Recipe.js
import Ingredients from "../Ingredients";

const Recipe = ({recipe}) => {
  return (
    <div>
      <span className="potion-headline">Recipe {recipe.name} by {recipe.brewer.name}</span>  <br></br>
      <Ingredients ingredients={recipe.ingredients} />
    </div>
  )
}

export default Recipe
